### ISSUE TITLE
LDA docs clarification

### ISSUE THREAD

**Issue by pdimens** [JuliaStats/MultivariateStats.jl#197]
The docs for LDA are quite technical but would benefit from some user-friendly explanations of the expected return values of a typical workflow including `predict(lda_results, newdata)` with the same level of detail as presented for the PCA section.

**Comment by wildart** [comment]
Yes, some of the docs are scarce, and in need of more content, e.g. examples & explanations. If you have something in mind, please, let us know or send updates.

**Comment by pdimens** [comment]
I'd love to, but I'm struggling a bit with the MC-LDA, which is why I bring this up =/

**Comment by wildart** [comment]
You can try to replicate PCA example with Iris dataset to show LDA dimetionality reduction mode, see this [scikit-learn example](
https://scikit-learn.org/stable/auto_examples/decomposition/plot_pca_vs_lda.html).

If you want to show how LDA is used as classifier, then it will be more complicated because the package doesn't provide any classifier. You can start splitting data on training and testing parts. Fit the training data to MC-LDA model (you can also reduce dimensionality, so you can visualize results). Next, you calculate predictions using training data, and feed the original data and the predicted one into the classifier, e.g. nearest neighbors.  Look at [MLBase.j](https://github.com/JuliaStats/MLBase.jl) for some classification primitives and performance evaluation functionality.

**Comment by pdimens** [comment]
Ah, I see. So `MulticlassLDA` does not provide the analagous functionality to the lda present in the R package `MASS`? I was trying to follow along [this interesting post](https://towardsdatascience.com/is-lda-a-dimensionality-reduction-technique-or-a-classifier-algorithm-eeed4de9953a) to see if it can be done with the present functionality. I suppose not. Unfortunately I don't have the intensive maths background to address this myself, but I will have a look at what MASS is doing out of curiosity.
